contracts/Liquidator/Liquidator.sol

```diff
@@ -381,17 +398,43 @@ contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, Liqu
         if (IVBep20(address(vTokenCollateral)).redeem(ours) != 0) {
             pendingRedeem.push(address(vTokenCollateral));
         } else {
-            address underlying = IVBep20(address(vTokenCollateral)).underlying();
-            uint256 underlyingBalance = IVToken(underlying).balanceOf((address(this)));
-            if (!IVToken(underlying).transfer(protocolShareReserve, underlyingBalance)) {
-                revert UnderlyingTransferFailed(address(vTokenCollateral), underlying);
+            if (address(vTokenCollateral) == address(vBnb)) {
+                _reduceBnbReserves();
+            } else {
+                _reduceVTokenReserves(address(vTokenCollateral));
             }
-            IProtocolShareReserve(protocolShareReserve).updateAssetsState(address(comptroller), underlying, 1);
-            emit ReservesReduced(msg.sender, address(vTokenCollateral), ours);
         }
         return (ours, theirs);
     }
 
+    function _reduceBnbReserves() private {
+        uint256 bnbBalance = address(this).balance;
+        IWBNB(wBNB).deposit{ value: bnbBalance }();
+        IWBNB(wBNB).withdraw(bnbBalance);
+        uint256 wBnbBalance = IWBNB(wBNB).balanceOf(address(this));
+        IWBNB(wBNB).transfer(protocolShareReserve, wBnbBalance);
+        IProtocolShareReserve(protocolShareReserve).updateAssetsState(
+            address(comptroller),
+            address(vBnb),
+            IncomeType.LIQUIDATION
+        );
+        emit ReservesReduced(msg.sender, address(wBNB), wBnbBalance);
+    }
+
+    function _reduceVTokenReserves(address vToken) private {
+        address underlying = IVBep20(vToken).underlying();
+        uint256 underlyingBalance = IERC20Upgradeable(underlying).balanceOf((address(this)));
+        if (!IERC20Upgradeable(underlying).transfer(protocolShareReserve, underlyingBalance)) {
```

Comment:
Would it be better if we use `SafeERC20` and `safeTranfer`? (as we are doing in the SwapRouter case [here](https://github.com/VenusProtocol/venus-protocol/blob/develop/contracts/Swap/SwapRouter.sol#L20))